src/one/nio/serial/Json.java

```diff
@@ -70,6 +75,18 @@ public static void appendObject(StringBuilder builder, Object obj) throws IOExce
         }
     }
 
+    public static void appendLong(StringBuilder builder, long value) {
```

Comment:
I believe writing long as string should be configurable (as in JACKSON/GSON)

Reply:
Fine with me.